src/WebJobs.Extensions.Http/Config/HttpExtensionConfigProvider.cs

```diff
@@ -14,13 +14,15 @@ namespace Microsoft.Azure.WebJobs.Extensions.Http
     internal class HttpExtensionConfigProvider : IExtensionConfigProvider
     {
         private readonly IOptions<HttpOptions> _options;
+        private readonly INameResolver _nameResolver;
 
         /// <summary>
         /// Constructs a new instance.
         /// </summary>
-        public HttpExtensionConfigProvider(IOptions<HttpOptions> options)
+        public HttpExtensionConfigProvider(IOptions<HttpOptions> options, INameResolver nameResolver)
         {
             _options = options;
+            _nameResolver = nameResolver;
```

Comment:
Revert changes to this file as it doesn't look like this field is needed anymore.